### PR TITLE
Configure pylint

### PR DESCRIPTION
**Pull request by ivan-podorozhnyi**
1. `requirements-dev.txt` file updated with required dependencies for pylint.
2. Add `.pylintrc` configuration file.
3. Add pylint trigger to verify_project.sh.

Affects #5

### CODE REVIEW

**Comment by ivan-podorozhnyi** [comment]
@extsoft This issue is not solved yet. pylint call in `verify_project.sh` exits with error code 24 and I can't find out the reason now. I will finish it soon.

**Comment by ivan-podorozhnyi** [comment]
@extsoft Please check my last commit. We can add pylint configuration file later if needed.

**Comment by ivan-podorozhnyi** [comment]
@extsoft I found the source of exit codes problem of pylint. Pylint exits with 0 exit code when our code is absolutely correct i.e. no warnings, errors etc. were found. 
For more details you can check https://docs.pylint.org/en/1.6.0/run.html section "Exit codes".
And in our current project pylint found black-listed name of module `foo.bar` and exits with exit code 16. That's why I added `--disable=all` flag. Pylint runs and gives no error exit code but in fact it checks nothing. So we need to change the name of this module.
In addition we will need to add `.pylintrc` configuration file to our project. Please let me know if it's ok. I can make changes accordingly.

**Comment by extsoft** [comment]
@ivan-podorozhnyi 
1. Please create `.pylintrc`if you think we need this file.
2. `--disable=all` means don't check. It's very bad idea. So, please remove this.
3. Close #14 and then continue with this.

P.S.
@ivan-podorozhnyi next time please ask such questions in the task, not in the pull request. I think you faced these questions before a pull request. That's why we have to discuss them not in the pull request.

**Comment by ivan-podorozhnyi** [comment]
@extsoft Changes applied. Please merge #15 before reviewing this pull request.

**Review by AntonKarabaza — APPROVED** [review]

File: verify_project.sh [hunk 1]
```diff
@@ -4,5 +4,7 @@ set -e
 py.test --cov=demo_python_at
 flake8 demo_python_at/
 pydocstyle --config=.pydocstyle demo_python_at
+pylint demo_python_at/
+pylint tests/
```
Comment (extsoft) [inline verify_project.sh]: @ivan-podorozhnyi we don't need to check unit tests.

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+# Create a graph of internal dependencies in the given file (report RP0402 must
+# not be disabled)
+int-import-graph=
+
+
+[EXCEPTIONS]
+
+# Exceptions that will emit a warning when being caught. Defaults to
+# "Exception"
+overgeneral-exceptions=Exception
```
Comment (extsoft) [inline .pylintrc]: @ivan-podorozhnyi please add an empty line at the end of the file.

**Review by extsoft — CHANGES_REQUESTED** [review]
@ivan-podorozhnyi please take o look for my comments.

**Comment by ivan-podorozhnyi** [comment]
@extsoft Unit tests check was removed and empty line was added at the end of `.pylintrc`. Please confirm.

**Review by extsoft — APPROVED** [review]

**Comment by extsoft** [comment]
@ivan-podorozhnyi accepted. Great job!

**Pull request merged**